cmd/dump-id/main.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 func main() {
+	// XXX: check that this still works!
```

Comment:
Not a big deal if it doesn't, it's already a bit nerfed from TUI refactoring. It used to give you a deep dump of the ID, but now it abbreviates into things like `Container.withExec` which isn't as useful.